src/main/scala/edu/berkeley/cs/rise/opaque/Utils.scala

```diff
@@ -106,6 +106,7 @@ import edu.berkeley.cs.rise.opaque.expressions.DotProduct
 import edu.berkeley.cs.rise.opaque.expressions.VectorAdd
 import edu.berkeley.cs.rise.opaque.expressions.VectorMultiply
 import edu.berkeley.cs.rise.opaque.expressions.VectorSum
+import edu.berkeley.cs.rise.opaque.expressions.Upper
```

Comment:
We should use Catalyst's Upper expression since it already exists. Upper shouldn't be a UDF.

Reply:
Changed to use catalyst upper